src/main/msp/msp_box.c

```diff
@@ -82,7 +82,7 @@ static const box_t boxes[CHECKBOX_ITEM_COUNT] = {
     { .boxId = BOXCAMERA1, .boxName = "CAMERA CONTROL 1", .permanentId = 32},
     { .boxId = BOXCAMERA2, .boxName = "CAMERA CONTROL 2", .permanentId = 33},
     { .boxId = BOXCAMERA3, .boxName = "CAMERA CONTROL 3", .permanentId = 34 },
-    { .boxId = BOXFLIPOVERAFTERCRASH, .boxName = "FLIP OVER AFTER CRASH", .permanentId = 35 },
+    { .boxId = BOXCRASHFLIP, .boxName = "FLIP OVER AFTER CRASH", .permanentId = 35 },
```

Comment:
When `boxName` is not updated there are no further configuration changes needed ?

Reply:
Also `FLIPMODE`

Reply:
crash flip sounds a bit more professional but im ok wit the turtle

Reply:
I think these strings are what the user sees in the modes tab.  The long text name `FLIP OVER AFTER CRASH` describes what the switch does.
In this case `TURTLE MODE` would indeed be more transparent, but this will do.
Maybe we can leave the change to `TURTLE` for another PR, unless we see more support for such a change.